README.md

```diff
@@ -79,7 +81,7 @@ The examples listed below use [catkin_tools](https://catkin-tools.readthedocs.io
 - Pull the packages into the repository:
   ```bash
   cd ~/path/to/tracebot_mockup_ws/src
-  git clone https://github.com/tecnalia-medical-robotics/tracebot_mockup.git
+  git clone https://github.com/BenRJG/tracebot_mockup.git # <=Branch Original => git clone https://github.com/tecnalia-medical-robotics/tracebot_mockup.git
```

Comment:
This should probably not go into this repository. Furthermore, if you change it, we can already replace the reference altogether with the soon to be default repository location, i.e. https://gitlab.com/tracebot/tracebot_mockup.git.